SharpPcap/LibPcap/LibPcapLiveDevice.cs

```diff
@@ -349,5 +365,57 @@ public override ICaptureStatistics Statistics
                 return new PcapStatistics(this.m_pcapAdapterHandle);
             }
         }
+
+#region Timestamp
+        /// <summary>
+        /// Timestamps supported by this device
+        /// </summary>
+        ///
+        /// <remarks>
+        /// Note: Live devices can have supported timestamps but offline devices
+        /// (such as file readers etc) do not. See https://www.tcpdump.org/manpages/pcap-tstamp.7.html
+        /// </remarks>
+        public System.Collections.Generic.List<PcapClock> TimestampsSupported
```

Comment:
I was thinking that this may go better in the `PcapInterface` class, instead of here. thoughts ?

Reply:
While the pcap pages do indicate that only capture devices support these calls I'm guessing, but haven't tested, that it won't have some exception. Let me see about moving it over. I can't say I have a good feel for whether its an improvement to do so yet.